examples/get_started/lv_example_get_started_2.c

```diff
@@ -31,4 +31,3 @@ void lv_example_get_started_2(void)
 }
 
 #endif
-
```

Comment:
Why delete the last line of the file? If it was deleted accidentally, please keep it.

Reply:
I though it's just a rebase issue as some empty lines were removed in an other PR. However rebasing locally  haven't resulted in any changes. 

Anyway, I agree to not remove the last empty lines in the PR.

Reply:
I did just run the scripts :P
The only actual change i did was in the CMake File :D